modules/dynamic-plugins/ref-example-third-party-plugin-installation.adoc

```diff
@@ -0,0 +1,160 @@
+[id="ref-example-third-party-plugin-installation_{context}"]
+= Example of installing a third-party plugin in {product}
+
+This section provides step-by-step instructions for integrating the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] into your {product-short}.
+
+. *Obtain the third-party plugin source code*: Clone the plugins repository and navigate to the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] directory:
++
+--
+.Obtain the third-party plugin source code
+[source,bash]
+----
+$ git clone https://github.com/backstage/community-plugins
+$ cd community-plugins/workspaces/todo
+$ yarn install
+----
+--
+
+. *Export backend and front-end plugins*: Run the following commands to build the backend plugin, adjust package dependencies for dynamic loading, and generate self-contained configuration schema:
++
+--
+.Export a backend plugin
+[source,bash]
+----
+$ cd todo-backend
+$ npx @janus-idp/cli@latest package export-dynamic-plugin
+----
+
+.Output of exporting a backend plugin commands
+[source,bash]
+----
+Building main package
+  executing     yarn build ✔
+Packing main package to dist-dynamic/package.json
+Customizing main package in dist-dynamic/package.json for dynamic loading
+  moving @backstage/backend-common to peerDependencies
+  moving @backstage/backend-openapi-utils to peerDependencies
+  moving @backstage/backend-plugin-api to peerDependencies
+  moving @backstage/catalog-client to peerDependencies
+  moving @backstage/catalog-model to peerDependencies
+  moving @backstage/config to peerDependencies
+  moving @backstage/errors to peerDependencies
+  moving @backstage/integration to peerDependencies
+  moving @backstage/plugin-catalog-node to peerDependencies
+Installing private dependencies of the main package
+   executing     yarn install --no-immutable ✔
+Validating private dependencies
+Validating plugin entry points
+Saving self-contained config schema in /Users/user/Code/community-plugins/workspaces/todo/plugins/todo-backend/dist-dynamic/dist/configSchema.json
+----
+
```

Comment:
Consider omitting the output (fluff + not maintainable).

```suggestion
```

Reply:
The output in the examples can be useful for users, to help them understand what is going on.

Reply:
@themr0c Since this is a reference module, we are providing examples for users to refer to.